### PR TITLE
Pydap version 3.5

### PR DESCRIPTION
**Pull request by Mikejmnez**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### additional notes

1) re-build the meta.yaml file with
```python
grayskull pypi pydap
```
2) Added myself as maintainer

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
 
 test:
   imports:
     - pydap
-  requires:
-    - gunicorn  # [not win]
-    - pastedeploy  # [not win]
-    - pip
-  commands:
-    - pydap -h  # [not win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:43]: Why did you remove the CLI test? Was it removed upstream? Or it was failing for some reason?

Comment (Mikejmnez) [reply recipe/meta.yaml:43]: Hmm I generated this with `grayskull`, and this was done automatically so I am not familiar with what does `-h` flags mean.

Comment (ocefpaf) [reply recipe/meta.yaml:43]: Grayskull helps but it is not a silver bullet. That only tests the CLI command. `-h` should output the help text.

Comment (Mikejmnez) [reply recipe/meta.yaml:43]: Oh I see. I though `-h` was some conda stuff for installing. It looks like grayskull left the `pydap` server testing.
```
pydap -h 
```
simply prints the `help docstring` when using pydap as a server. Will add those now

**Comment by ocefpaf** [comment]
Thanks @Mikejmnez!

**Pull request merged**